src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.c

```diff
@@ -22,50 +30,219 @@ const ActorInit Obj_Kibako2_InitVars = {
     (ActorFunc)ObjKibako2_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_8098EE60 = {
+static ColliderCylinderInit sCylinderInit = {
     { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_TYPE_2, COLSHAPE_CYLINDER, },
     { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x80000508, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
     { 31, 48, 0, { 0, 0, 0 } },
 };
```

Comment:
This doesn't seem formatted properly, did you run `format.sh`? If you did make sure that you have clang-format-11 installed, otherwise the formatting will fail.

Reply:
I did run `format.sh`, but I didn't have `clang-format-11` installed. I was wondering why the `format.sh` didn't seem to do anything; I guess that explains it. Fixed now, thanks for pointing it out.